Formula/qt.rb

```diff
@@ -47,6 +47,7 @@ class Qt < Formula
   depends_on "libpng"
   depends_on "libtiff"
   depends_on "md4c"
+  depends_on "molten-vk"
```

Comment:
Is this a runtime or a build time dependency?

Reply:
Both